packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -563,6 +563,101 @@ const defaultConfigs = {
     uniswapAddress: "0xedf187890af846bd59f560827ebd2091c49b75df",
     twapLength: 7200,
     invertPrice: true
+  },
+  USDAAVE: {
```

Comment:
Think you're going to need a `lookback` on any pricefeed that uses a Cryptowatch pricefeed.

Can set it under `invertPrice`: `lookback: 7200`, similar to `USDETH`